binding/default_validator_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Gin Core Team. All rights reserved.
+// Use of this source code is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+package binding
+
+import (
+	"errors"
+	"testing"
+)
+
+func TestSliceValidateError_Error(t *testing.T) {
```

Comment:
please rename using `TestSliceValidateError`

Reply:
fixed in 8098d88562163a827f495e72b49bb936f0ebe881

Reply:
> please rename using `TestSliceValidateError`

fixed in 03ce1a6b864de65c92f305e8b6d90c1675f2c9e2